components/CreateEventForm/CreateEventForm.js

```diff
@@ -32,12 +32,12 @@ export function CreateEventForm(props) {
   }
 
   const onCreateEventPress = () => {
-    // if (date.getTime() < Date.now()) {
-    //   return Alert.alert("Sorry, but currently we don't allow events to be created on the day of the event or prior to today's date.")
-    // }
     if (equipmentRequired.split(',').length > 5) {
       return Alert.alert('Please limit your equipment to 5 items.')
     }
+    if (nameOfActivity === "Custom" && !notes) {
+      return Alert.alert("Please enter the name of your custom event in the 'Notes' section above.")
+    }
```

Comment:
It looks like this code is no longer relevant since we're handling it as conditionally rendered <Text> tag. It should be removed.